README.md

```diff
@@ -10,91 +10,93 @@
 # Cartes
 
 Ce système est conçu pour deux cartes :
-- Une carte réalisée pour Maker Faire 2018, qui ne fonctionne qu'avec les Nabaztag v1 (sans micro ni RFID).
-- Une nouvelle version de la carte, proposée via la campagne Ulule en mai 2019, qui fonctionne avec les Nabaztag v1 et v2 (les micros sont sur la carte, du coup les Nabaztag v1 bénéficient aussi de la reconnaissance vocale).
+- Une carte réalisée pour [Maker Faire 2018](https://paris.makerfaire.com/maker/entry/1285/), qui ne fonctionne qu'avec les Nabaztag v1 (sans micro ni RFID).
+- Une nouvelle version de la carte, proposée via la [campagne Ulule en mai 2019](https://fr.ulule.com/le-retour-du-nabaztag/), qui fonctionne avec les Nabaztag v1 et v2 (les micros sont sur la carte, du coup les Nabaztag v1 bénéficient aussi de la reconnaissance vocale).
 
 Les schémas et fichiers de fabrication de ces deux cartes sont dans le repository [hardware](https://github.com/nabaztag2018/hardware), respectivement [`RPI_Nabaztag`](https://github.com/nabaztag2018/hardware/blob/master/RPI_Nabaztag.PDF) (2018) et [`tagtagtag_V2.0`](https://github.com/nabaztag2018/hardware/tree/master/tagtagtag_V2.0) (2019).
 
+Pour être prévenu de la prochaine campagne, vous pouvez vous inscrire à la [Mailing list](https://tinyletter.com/nabaztag).
+
 # Images
 
 Les [releases](https://github.com/nabaztag2018/pynab/releases) sont des images de Raspbian Buster Lite 2019-09-26 avec pynab pré-installé. Elles ont les mêmes réglages que [Raspbian](https://www.raspberrypi.org/downloads/raspbian/).
 
-Les releases actuelles (0.7.x) ne fonctionnent que sur les cartes 2019 (cf #44)
+Les releases actuelles (>0.7.x) ne fonctionnent que sur les cartes 2019 (cf [#44](https://github.com/nabaztag2018/pynab/issues/44)).
 
 # Installation sur Raspbian (pour développeurs !)
 
 0. S'assurer que le système est bien à jour
 
 Le script d'installation requiert désormais une Raspbian avec buster, pour bénéficier de Python 3.7.
-Il est nécessaire que les headers depuis le paquet apt correspondent à la version du noyau.
+Il est nécessaire que les headers depuis le paquet `apt` correspondent à la version du noyau.
 
-```
+```sh
 sudo apt update
 sudo apt upgrade
 ```
 
 1. Configurer la carte son, les oreilles et le lecteur RFID et redémarrer.
 
-Maker Faire 2018 :
-https://support.hifiberry.com/hc/en-us/articles/205377651-Configuring-Linux-4-x-or-higher
-
-Ulule 2019 :
-https://github.com/pguyot/wm8960/tree/tagtagtag-sound
+Carte son :
```

Comment:
C'est confus, non ?

Dans le texte initial, on distinguait les cartes Maker Faire 2018 et les cartes Ulule 2019. Certes les premières ont des cartes son séparées, mais dans le (1) Configurer la carte son, c'est un terme générique (chipset + électronique associée). On pourrait mettre "Configurer le son".

Pour les cartes Maker Faire 2018, il faut le driver hifiberry.
Pour les cartes Ulule 2019, il faut le driver tagtagtag-sound.

Reply:
En effet, le texte initial était plus clair sur la spécificité du driver son suivant le type de carte.

Reply:
I updated the readme 😉